src/pwncore/routes/ctf/start.py

```diff
@@ -144,13 +141,13 @@ async def stopall_docker_container(response: Response):
 
 @atomic()
 @router.post("/stop/{ctf_id}")
-async def stop_docker_container(ctf_id: int, response: Response):
+async def stop_docker_container(ctf_id: int, response: Response, jwt=Depends(get_jwt)):
```

Comment:
change the style here too

Reply:
As mentioned here: https://github.com/lugvitc/pwncore/pull/6#discussion_r1404238694